src/engine/source/api/src/kvdb/handlers.cpp

```diff
@@ -134,8 +160,20 @@ api::Handler managerDump(std::shared_ptr<kvdb_manager::KVDBManager> kvdbManager)
         {
             return ::api::adapter::genericError<ResponseType>("Missing /name");
         }
+        if (eRequest.name() == "")
+        {
+            return ::api::adapter::genericError<ResponseType>("Database name is empty");
```

Comment:
Please, keep the same messages format, it seems that in the rest of the messages, field names are written with the `/` prefix

Reply:
The same goes for all the occurrences